.github/workflows/deploy.yml

```diff
@@ -21,4 +21,6 @@ jobs:
             cd /var/www/share/devdocs.mage-os.org/devdocs
             ./bin/checkout_latest_docs.sh
             php artisan optimize
-            php artisan sitemap:generate
\ No newline at end of file
+            php artisan optimize:clear
```

Comment:
Do we really need to call this twice, or could we first generate the sitemap and then only clear once?

Reply:
tbh, I haven't dug into it, but it solves the deploy issue. cba spending more time into that for now. Its good enough

Reply:
Isn't this a very simple test? I mean, we just need to check if we can remove the first clear, right? But okay :shrug: